src/liballoc_system/lib.rs

```diff
@@ -39,29 +39,35 @@ const MIN_ALIGN: usize = 8;
 const MIN_ALIGN: usize = 16;
 
 #[no_mangle]
-pub extern fn __rust_allocate(size: usize, align: usize) -> *mut u8 {
+pub extern "C" fn __rust_allocate(size: usize, align: usize) -> *mut u8 {
```

Comment:
As a fan of long dead https://github.com/rust-lang/rfcs/pull/697 I appreciate this little initiative from rustfmt.